feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go

```diff
@@ -31,6 +31,7 @@ var (
 	npDesc       = "New Primary"
 	gdpEtherType = *ygot.Uint32(0x6007)
 	portId       = *ygot.Uint32(10)
+	if_type      = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
```

Comment:
remove the variable and use it directly while building the struct.

Reply:
I have made the changes.